std/meta.d

```diff
@@ -1317,9 +1317,12 @@ private template SmartAlias(T...)
  * Creates an `AliasSeq` which repeats a type or an `AliasSeq` exactly `n` times.
  */
 template Repeat(size_t n, TList...)
-if (n > 0)
```

Comment:
Please add a `n >= 0` template constraint.

Reply:
no need, size_t is unsigned

Reply:
Whoops, missed that.